### DISCUSSION TITLE
Write-Ups & Reports

### DISCUSSION THREAD

**Discussion by roinevirta** [roinevirta/euro-stablecoin-research#21] [Resources]
Write-Ups & Reports on stablecoins. 

Please add automatically updating, or recurrently updated, datasets to https://github.com/roinevirta/euro-stablecoin-research/discussions/19

**Comment by roinevirta** [comment]
**Title**: Stablecoins Report: The Ultimate 2023 Market Overview
**Author**: RedStone Finance
**Date**: 2023
**Level**: Novice
**TL;DR**: Market overview of some stablecoins, basics between different system designs
**Euro stablecoins**: agEUR
**Available at**: [https://redstone.finance/reports/stablecoins-report.pdf](https://redstone.finance/reports/stablecoins-report.pdf)

**Comment by roinevirta** [comment]
**Title**: LlamaRisk Stablecoin Risk Assessment
**Author**: LlamaRisk
**Date**: Variable
**TL;DR**: Risk assessments on various stablecoins
**Euro stablecoins**: agEUR, EURS, EUROe, EURe
**Available at**: https://cryptorisks.substack.com/

**Comment by jeanbapt** [comment]
New kid in town from Germany : [https://allunity.com/](https://allunity.com/)
BaFin regulated 
Just a landing page. 
To put in the watch list.

**Reply by roinevirta** [reply]
The [issues](https://github.com/roinevirta/euro-stablecoin-research/issues) section contains more stablecoins that haven’t made it to the repo as well! :)
